frontend/src/metabase/query_builder/components/ExpressionPopover.jsx

```diff
@@ -52,7 +53,7 @@ export default class ExpressionPopover extends React.Component {
             onCommit={expression => {
               if (!onChangeName) {
                 onChange(expression);
```

Comment:
Since you're setting state in `onUpdateAndCommit` I don't think this `onChange(expression)` call is needed now.

Reply:
Oops, good catch! I'll remove that.